pkg/controlplane/storageversionhashdata/data.go

```diff
@@ -73,7 +73,8 @@ var GVRToStorageVersionHash = map[string]string{
 	"networking.k8s.io/v1beta1/ingressclasses":                             "l/iqIbDgFyQ=",
 	"networking.k8s.io/v1/ingresses":                                       "ZOAfGflaKd0=",
 	"networking.k8s.io/v1/ingressclasses":                                  "l/iqIbDgFyQ=",
-	"node.k8s.io/v1beta1/runtimeclasses":                                   "8nMHWqj34s0=",
+	"node.k8s.io/v1/runtimeclasses":                                        "WQTu1GL3T2Q=",
```

Comment:
I'm not sure how this hash is calculated. This one works =)

Reply:
this should change back, since we're still storing in v1beta1 format.